lua/exporters/object.lua

```diff
@@ -0,0 +1,54 @@
+--
+-- Converts DCS tables in the Object hierarchy into tables suitable for
+-- serialization into GRPC responses
+-- Each exporter has an equivalent .proto Message defined and they must
+-- be kept in sync
+--
+
+local GRPC = GRPC
+local coord = coord
+
+local function toLatLonPosition(pos)
+  local lat, lon, alt = coord.LOtoLL(pos)
+  return {
+    lat = lat,
+    lon = lon,
+    alt = alt,
+  }
+end
+
+GRPC.exporters.unit = function(unit)
+  local u = {}
+  u["id"] = unit:getID()
+  u["name"] = unit:getName()
+  u["callsign"] = unit:getCallsign()
+  u["coalition"] = unit:getCoalition();
+  u["type"] = unit:getTypeName()
+  u["position"] = toLatLonPosition(unit:getPoint())

```

Comment:
Any reason to not assign anything at once, ala:

```lua
local u = {
  id = unit:getID(),
  ...
}
```

Or mainly personal preference? I don't have a strong opinion here either, so the question is mostly about being curious.

Reply:
No proper reason. I am still fumbling around in lua (god I do not enjoy writing lua...)

Switched to the suggested layout.